jib-core/src/main/java/com/google/cloud/tools/jib/docker/ImageTarball.java

```diff
@@ -42,16 +43,20 @@
 
   private final Image image;
   private final ImageReference imageReference;
+  private final ImmutableSet<String> allTargetImageTags;
 
   /**
    * Instantiate with an {@link Image}.
    *
    * @param image the image to convert into a tarball
    * @param imageReference image reference to set in the manifest
```

Comment:
I'd like to make it clear that the tag portion of this is ignored, and you'll get nothing if you pass an empty set for `allTargetImageTags`.

Or, we could accept `additionalTargetImageTags` instead, and do

```java
this.allTargetImageTags = new Set of imageReference.getTag() + additionalTargetImageTags
```